<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerGamingWidget</class>
 <widget class="QWidget" name="playerGamingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="wordLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>110</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="wordLineEdit">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>200</y>
     <width>120</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>140</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>170</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="levelLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡</string>
   </property>
  </widget>
  <widget class="QLabel" name="passLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x/y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回菜单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>195</y>
     <width>41</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
